OneSignalSDK/onesignal/src/main/java/com/onesignal/GooglePlayServicesUpgradePrompt.java

```diff
@@ -30,8 +30,10 @@ private static boolean isGooglePlayStoreInstalled() {
          PackageManager pm = OneSignal.appContext.getPackageManager();
          PackageInfo info = pm.getPackageInfo(GoogleApiAvailability.GOOGLE_PLAY_SERVICES_PACKAGE, PackageManager.GET_META_DATA);
          String label = (String) info.applicationInfo.loadLabel(pm);
-         return (label != null && !label.equals("Market"));
-      } catch (Throwable e) {}
+         return (!label.equals("Market"));
```

Comment:
is it ok to remove the != null check?
In that case the ()  can be removed

Reply:
@Jeasmine Good eye, however the return definition of `PackageInfo.loadLabel()` has a `@NonNull`.

```java
public @NonNull CharSequence loadLabel(@NonNull PackageManager pm) {
```